### PR TITLE
feat(ClassOverview): link :hover

### PR DESCRIPTION
**Pull request by zacimac**
Something I kinda rely on... Mainly because I'm just bad at using a mouse.

Before
Before GIF

After
After GIF

### CODE REVIEW

File: src/components/ClassOverview.vue [hunk 1]
```diff
@@ -19,12 +19,12 @@
 					<li
 						v-for="property in visibleProperties"
 						:key="scopedName(property)"
-						class="cursor-pointer mb-14"
+						class="cursor-pointer mb-14 border-l-4 border-transparent hover:border-l-4 hover:border-discord-blurple-500"
```
Comment (iCrawl) [inline src/components/ClassOverview.vue:22]: Can we add a `rounded-sm` to this?

File: src/components/ClassOverview.vue [hunk 2]
```diff
@@ -74,12 +74,12 @@
 					<li
 						v-for="method in visibleMethods"
 						:key="scopedName(method)"
-						class="cursor-pointer"
+						class="cursor-pointer border-l-4 border-transparent hover:border-l-4 hover:border-discord-blurple-500"
```
Comment (iCrawl) [inline src/components/ClassOverview.vue:77]: See above.

File: src/components/ClassOverview.vue [hunk 3]
```diff
@@ -129,12 +129,12 @@
 					<li
 						v-for="event in visibleEvents"
 						:key="event.name"
-						class="cursor-pointer"
+						class="cursor-pointer border-l-4 border-transparent hover:border-l-4 hover:border-discord-blurple-500"
```
Comment (iCrawl) [inline src/components/ClassOverview.vue:132]: See above.

**Review by iCrawl — CHANGES_REQUESTED** [review]

**Comment by zacimac** [comment]
All rounded! :)

File: src/components/ClassOverview.vue [hunk 4]
```diff
@@ -19,12 +19,18 @@
...
+							border-l-4 border-transparent
+							rounded-sm
+							hover:border-l-4 hover:border-discord-blurple-500
+						"
 						@click="scrollTo(scopedName(property))"
 					>
 						<router-link
 							:to="{ name: 'docs-source-tag-class-class', query: { scrollTo: scopedName(property) } }"
-							class="mr-2"
+							class="pl-1 mr-2"
```
Comment (iCrawl) [inline src/components/ClassOverview.vue:33]: One more thing I noticed in the preview, can we make this `mx-2` instead? (And remove the `pl-1`)

File: src/components/ClassOverview.vue [hunk 5]
```diff
@@ -74,12 +80,17 @@
...
+							border-l-4 border-transparent
+							rounded-sm
+							hover:border-l-4 hover:border-discord-blurple-500
+						"
 						@click="scrollTo(scopedName(method))"
 					>
 						<router-link
 							:to="{ name: 'docs-source-tag-class-class', query: { scrollTo: scopedName(method) } }"
-							class="mr-2"
+							class="ml-1 mr-2"
```
Comment (iCrawl) [inline src/components/ClassOverview.vue:93]: See above.

File: src/components/ClassOverview.vue [hunk 6]
```diff
@@ -129,12 +140,17 @@
...
+							border-l-4 border-transparent
+							rounded-sm
+							hover:border-l-4 hover:border-discord-blurple-500
+						"
 						@click="scrollTo(`e-${event.name}`)"
 					>
 						<router-link
 							:to="{ name: 'docs-source-tag-class-class', query: { scrollTo: `e-${event.name}` } }"
-							class="mr-2"
+							class="ml-1 mr-2"
```
Comment (iCrawl) [inline src/components/ClassOverview.vue:153]: See above.

**Review by iCrawl — CHANGES_REQUESTED** [review]

File: src/components/ClassOverview.vue [hunk 7]
```diff
@@ -30,7 +30,7 @@
 					>
 						<router-link
 							:to="{ name: 'docs-source-tag-class-class', query: { scrollTo: scopedName(property) } }"
-							class="pl-1 mr-2"
+							class="mx-2 mr-2"
```
Comment (iCrawl) [inline src/components/ClassOverview.vue:33]: I may have explained this badly for someone now really knowing tailwindcss, sorry.

You need to remove `mr-2` here now that `mx-2` is a thing. `mx-2` is a shortcut for `ml-2 mr-2`

File: src/components/ClassOverview.vue [hunk 8]
```diff
@@ -90,7 +90,7 @@
 					>
 						<router-link
 							:to="{ name: 'docs-source-tag-class-class', query: { scrollTo: scopedName(method) } }"
-							class="ml-1 mr-2"
+							class="mx-2 mr-2"
```
Comment (iCrawl) [inline src/components/ClassOverview.vue:93]: See above.

File: src/components/ClassOverview.vue [hunk 9]
```diff
@@ -150,7 +150,7 @@
 					>
 						<router-link
 							:to="{ name: 'docs-source-tag-class-class', query: { scrollTo: `e-${event.name}` } }"
-							class="ml-1 mr-2"
+							class="mx-2 mr-2"
```
Comment (iCrawl) [inline src/components/ClassOverview.vue:153]: See above.

**Review by iCrawl — CHANGES_REQUESTED** [review]

File: src/components/ClassOverview.vue (hunk 7 above)
Comment (zacimac) [reply src/components/ClassOverview.vue:33]: My bad 🤦‍♂️

**Comment by iCrawl** [comment]
Perfect, thank you!

**Pull request merged**